### PR TITLE
bump api and istio refs

### PR DESCRIPTION
**Pull request by dhawton**
## Description

Resolve lint failure due to changed api

## Reviewers

- [ ] Ambient
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [ ] Localization/Translation

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -195,7 +195,7 @@ require (
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 	helm.sh/helm/v3 v3.14.4 // indirect
-	istio.io/api v1.22.0-beta.0 // indirect
```
Comment (craigbox) [inline go.mod:198]: are these sensible changes? reads like a move backwards

**Review by craigbox — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (dhawton) [reply go.mod:198]: beta.0's tag is actually older than 09344006f961. So while it looks odd "going back" to alpha, the commit used is newer. I don't know how Go does tags like this.

**Comment by dhawton** [comment]
/test doc.test.profile-none

This is an odd failure.. the same bookinfo startup func is used in other tests without issue...

**Comment by craigbox** [comment]
/test doc.test.profile-none

I can do this all day.

**Review by kfaseela — APPROVED** [review]

**Comment by kfaseela** [comment]
@zirain any idea why the tracing test fails?

**Comment by kfaseela** [comment]
Does not look like the usual flakes

**Comment by dhawton** [comment]
> @zirain any idea why the tracing test fails?

There was 1 or 2 runs where it was something else... on this, it is productpage-v1 timing out deploying, for some reason, even though the same deployment succeeds elsewhere. I don't see any info in the other logs to know why it's not deploying.

**Comment by dhawton** [comment]
Some branch protections now exist so I can't push to this branch anymore... reopening from fork.

**Pull request closed without merging**